### PR TITLE
Add Firefox update.rdf; Point to update.rdf in package.json

### LINKED ISSUE #643: Host Firefox Extension as Unlisted
Evals are prohibited and Mozilla isn't reviewing us.
- [ ] Create update.rdf
- [ ] Put update.rdf in a place where we will never move it

### PR DESCRIPTION
**Pull request by hobinjk**
Fixes #643

### CODE REVIEW

File: Firefox/update.rdf [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+
+        <!-- Each li is a different version of the same add-on -->
+        <RDF:li>
+          <RDF:Description>
+            <em:version>7.7.1</em:version> <!-- This is the version number of the add-on -->
+
+            <!-- One targetApplication for each application the add-on is compatible with -->
+            <em:targetApplication>
+              <RDF:Description>
+                <em:id>{ec8030f7-c20a-464f-9b0e-13a3a9e97384}</em:id>
```
Comment (Wolvan) [inline Firefox/update.rdf]: Looks like we only specify Firefox here, Fennec (Firefox for Android) needs to be added as well afaik

Comment (hobinjk) [inline Firefox/update.rdf]: Good point, I forgot to include that UUID

File: Firefox/update.rdf [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+                <!-- A page describing what is new in this updated version -->
+                <em:updateInfoURL>http://new-xkit-extension.tumblr.com/post/130919884664/new-xkit-771-release-firefox-users-must</em:updateInfoURL>
+              </RDF:Description>
+            </em:targetApplication>
+
+            <!-- Support Fennec (Firefox for Android) as well -->
+            <em:targetApplication>
+              <RDF:Description>
+                <em:id>{aa3c5121-dab2-40e2-81ca-7ea25febc110}</em:id>
+                <em:minVersion>35.0</em:minVersion>
```
Comment (Wolvan) [inline Firefox/update.rdf]: just wanna make sure, are the fennec versions the same as firefox?

Comment (hobinjk) [inline Firefox/update.rdf]: I have no reason to believe that they should be different. This is based off of the `install.rdf` generated by JPM

Comment (Wolvan) [inline Firefox/update.rdf]: alright, fair enough

**Comment by Wolvan** [comment]
@homu r+

**Pull request merged**